frictionless/formats/sql/mapper.py

```diff
@@ -252,15 +269,20 @@ def write_field(self, field: Field, *, table_name: str) -> Column:  # type: igno
                     enum_name = "%s_%s_enum" % (table_name, field.name)
                     column_type = sa.Enum(*value, name=enum_name)
 
-        # Create column
-        column_args = [field.name, column_type] + checks  # type: ignore
+        # Prepare column
         # TODO: shall it use "autoincrement=False"
         # https://github.com/Mause/duckdb_engine/issues/595#issuecomment-1495408566
-        column_kwargs = {"nullable": nullable, "unique": unique}
+        column_args = [field.name, column_type]  # type: ignore
+        column_kwargs = {}
         if field.description:
             column_kwargs["comment"] = field.description
-        column = sa.Column(*column_args, **column_kwargs)
+        if not without_constraints:
+            column_args += checks  # type: ignore
+            column_kwargs["nullable"] = nullable
+            column_kwargs["unique"] = unique
```

Comment:
Nitpick: For better readability, maybe I would bring the definitions of checks, nullable and unique in this block (under the condition `if not without_constraints`). This has the advantage of dealing with constraints at a single place and not assigning variables that are not used.